release-notes/Cloud/DCP_change_log_2025.md

```diff
@@ -9,6 +9,14 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+### 28 January 2025 - New feature - Catalog API - Deploying a Catalog item using an organization key [ID 41987]
+
+It is now possible to deploy a Catalog item to a cloud connected system using an organization key.
+
+### 28 January 2025 - Fix - Catalog API - Registration failed because of invalid artifact references in Catalog versions [ID 41985]
```

Comment:
Do you mean "Catalog item versions", or is it indeed the version of the Catalog itself?

Reply:
its indeed about catalog item versions, updated the text.